examples/research_projects/wav2vec2/run_training.py

```diff
@@ -0,0 +1,220 @@
+#!/usr/bin/env python3
+from dataclasses import dataclass
+from typing import Any, Dict, List, Optional, Union
+
+import datasets
+import numpy as np
+import torch
+import torch.nn as nn
+from packaging import version
+
+import soundfile as sf
+from jiwer import wer
+from transformers import Trainer, TrainingArguments, Wav2Vec2ForCTC, Wav2Vec2Processor, is_apex_available
+
+
+if is_apex_available():
+    from apex import amp
+
+
+if version.parse(torch.__version__) >= version.parse("1.6"):
+    _is_native_amp_available = True
+    from torch.cuda.amp import autocast
+
+
+model = Wav2Vec2ForCTC.from_pretrained("facebook/wav2vec2-base", gradient_checkpointing=True)
+processor = Wav2Vec2Processor.from_pretrained("facebook/wav2vec2-base")
```

Comment:
The README said it was for any wav2vec checkpoint, so make `""facebook/wav2vec2-base"` an argument (the ususal model_name_or_path)?